rfcs/rfc-manage-staff-role-and-account.md

```diff
@@ -0,0 +1,75 @@
+- Start Date: (2022-07-23)
+- Target Major Version: N/A
+- Reference Issues: (fill in existing related issues, if any)
+- Entity Issue: https://github.com/bcgov/entity/issues/12226
+- Implementation PR: None
+
+
+# Summary
+
+SBC staff need to access the staff dashboard with limited privileges to support SP/GP filing/alteration.
+
+This is an RFC draft to discuss more suitable approaches 
+
+# Basic example
+
+If the proposal involves a new or changed API, component, etc., include a basic code example.
+Omit this section if it's not applicable.
+
+# Motivation
+SBC staff want to access the staff dashboard to search for SP/GP filing for clients, They may have to register SP/GP companies on behalf of the client
```

Comment:
Currently all systems have to _fake_ out an account # for staff, this is handled as an exception which in general we don't like.
Other agencies (like sbc staff, GA, call-center, etc)) also run into these same problems, a suggested long term fix is to remove the exceptions and hard-coding of fake accounts and make these like all other Auth users.